src/vs/workbench/parts/search/electron-browser/search.contribution.ts

```diff
@@ -139,6 +148,15 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
 	}
 });
 
+MenuRegistry.appendMenuItem(MenuId.SearchContext, {
+	command: {
+		id: Constants.ReplaceActionId,
```

Comment:
It is strange that this are called ActionId when you are actually registering a command

Reply:
Yes... will clean this up with the above debt item.

Reply:
Makes sense